### PR TITLE
Refactor _helper.scss

### PR DESCRIPTION
**Pull request by Jp138233**
## Description
Split _helper.scss into 12 different scss files, each containing a subset of the original scss.

## Related issues and discussion
#60 

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [check] If you have multiple commits please combine them into one commit by squashing them.

- [check] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/assets/styles/_align.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+body, html {
+  position: relative;
+  overflow: hidden;
+}
+
+  .v-mid,.v-m {
```
Comment (abhaychawla) [inline src/assets/styles/_align.scss]: Remove this extra space from each file wherever it has occurred and ensure proper indentation.

File: src/assets/styles/_align.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+body, html {
+  position: relative;
+  overflow: hidden;
+}
```
Comment (abhaychawla) [inline src/assets/styles/_align.scss]:
```css
html,
body {
  position: relative;
  overflow: hidden;
}
```

Change the syntax into this kind at all the places.

File: src/assets/styles/_background.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    background-color: white
+  }
+
+  .bg-none,.no-bg {
+    background: none;
+    background-image: none;
+    background-color: transparent
+  }
+
+  .bg-facebook {
```
Comment (abhaychawla) [inline src/assets/styles/_background.scss]: Remove this class. Its not required.

File: src/assets/styles/_background.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    background-color: #47639E
+  }
+
+  .bg-twitter {
```
Comment (abhaychawla) [inline src/assets/styles/_background.scss]: Remove this class. Its not required.

File: src/assets/styles/_align.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  .v-mid,.v-m {
+    vertical-align: middle !important
```
Comment (abhaychawla) [inline src/assets/styles/_align.scss]: Ensure there is a semi-colon (;) after every property in the class. Make the required changes in all files necessary.

File: src/assets/styles/_clear.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+  .clear {
```
Comment (abhaychawla) [inline src/assets/styles/_clear.scss]: Remove this file and its import.

File: src/assets/styles/_helper.scss [hunk 7]
```diff
@@ -2,1636 +2,40 @@
 @import "loader";
```
Comment (abhaychawla) [inline src/assets/styles/_helper.scss]: Arrange all imports in alphabetical order.

File: src/assets/styles/_helper.scss [hunk 8]
```diff
@@ -2,1636 +2,40 @@
...
 @import "width-height";
 @import "z-index";
+//Import helper files
```
Comment (abhaychawla) [inline src/assets/styles/_helper.scss]: Remove this comment.

File: src/assets/styles/_helper.scss [hunk 9]
```diff
@@ -2,1636 +2,40 @@
...
+@import "border";
+@import "background";
+@import "align";
+@import "clear";
+@import "float";
+@import "overflow";
+@import "position";
+
+
+//Miscellaneous scss
```
Comment (abhaychawla) [inline src/assets/styles/_helper.scss]: Create a separate file for miscellaneous scss.

File: src/assets/styles/_input.scss [hunk 10]
```diff
@@ -0,0 +1,24 @@
+  .full-input input,.full-input select,.full-input textarea {
```
Comment (abhaychawla) [inline src/assets/styles/_input.scss]: Rename file to `_form.scss`.

**Comment by abhaychawla** [comment]
@Jp138233 After making the required changes, run `stylelint "src/assets/styles/*.scss" --syntax scss` and make sure all lint errors are removed.

File: src/assets/styles/_clear.scss (hunk 6 above)
Comment (Jp138233) [reply src/assets/styles/_clear.scss]: Should the scss in the file be moved to another file or should it be removed as well?

Comment (abhaychawla) [reply src/assets/styles/_clear.scss]: Remove the scss as well, it is not required.

**Comment by abhaychawla** [comment]
@Jp138233 Have you made all the required changes and made sure all lint errors were removed? Also, use `x` instead of writing `check` within the boxes of the PR description (and make sure you follow the checklist) I see two commits right now.

**Comment by abhaychawla** [comment]
Closing this PR due to a long period of inactivity.

**Pull request closed without merging**